docs/DeveloperGuide.md

```diff
@@ -484,23 +500,34 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
    Use case resumes at step 2.
 
-* 3c. The given index is invalid.
+* 3c. The given command format is invalid.
+
+    * 3c1. Tuteez shows invalid command format error message.
+
+    Use case resumes at step 2.
+
+* 3c. The given student index is invalid.
 
-   * 3c1. Tuteez shows an error message.
+   * 3c1. Tuteez shows an invalid student error message.
```

Comment:
Perhaps this could be changed to "The system informs the user of an invalid student error" instead, to avoid mentioning UI details in our use cases